less/home-page.less

```diff
@@ -58,4 +58,22 @@
       margin-bottom: 30px;
     }
   }
-}// home
\ No newline at end of file
+}// home
+
+
+// MY CODE STARTS HERE
+
+.img-and-p {
+  margin-left: 34%;
```

Comment:
Instead of manually setting left and right margins, if you want to center something left and right you can use
```margin: 0 auto;```

In order to give the header the proper width to match the main content, there are two properties I recommend: `max-width` and `width`. width will always be 100%, and max-width will be set to the maximum size you want the container to have. (No max-width, and the header will be super duper wide in desktop view)

For this project, I looked into the `global.less` file and used these two values"
```
max-width: 800px;
width: 100%;
```
Adding that in should make the header fit better on the page 👍